events/index.html

```diff
@@ -9,10 +9,34 @@
         <!-- begin Events -->
         <hr class="whitespace">
         <div id="community-posts" class="row">
+            <div class="small-12 medium-8 medium-offset-2 columns">
+                <h6><i class="fa fa-calendar"></i> November 30 and December 2, 2020</h6>
+                <h2><a href="workshops-i2k-janelia-november-2020.html">OME Workshops, I2K virtual conference</a></h2>
+                <p class="card-caption">Four 4-hour demos/workshops on "Next-generation file formats" and "OMERO" held remotely.</p>
+            </div>
+            <hr class="medium-8">
+            <div class="small-12 medium-8 medium-offset-2 columns">
+                <h6><i class="fa fa-calendar"></i> November 27, 2020</h6>
+                <h2><a href="omero-users-workshop-gothenburg-online-november-2020.html">OMERO Workshop, University of Gothenburg</a></h2>
+                <p class="card-caption">A 2-hour demo/workshop on how to use OMERO held remotely.</p>
+            </div>
+            <hr class="medium-8">
+            <div class="small-12 medium-8 medium-offset-2 columns">
+                <h6><i class="fa fa-calendar"></i> November 18, 2020</h6>
+                <h2><a href="omero-users-workshop-montreal-online-november-2020.html">OMERO Workshop, McGill University, Montreal</a></h2>
+                <p class="card-caption">A 1-hour demo/workshop on how to use OMERO held remotely.</p>
+            </div>
+            <hr class="medium-8">
+            <div class="small-12 medium-8 medium-offset-2 columns">
+                <h6><i class="fa fa-calendar"></i> November 18, 2020</h6>
+                <h2><a href="omero-users-workshop-eastbio-edinburgh-online-november-2020.html">OMERO Workshop, EastBio, held remotely</a></h2>
+                <p class="card-caption">A 45 minutes demo/workshop on how to use OMERO.</p>
+            </div>
+            <hr class="medium-8">
             <div class="small-12 medium-8 medium-offset-2 columns">
                 <h6><i class="fa fa-calendar"></i> May 26-28, 2020</h6>
                 <h2><a href="{{ site.baseurl }}/events/ome-community-meeting-2020/">2020 OME Community Meeting</a></h2>
-                <p class="card-caption">The 2020 OME Community Meeting will be held remotely from May 26th to May 28th 2020.</p>
+                <p class="card-caption">The fdsafdsfa 2020 OME Community Meeting will be held remotely from May 26th to May 28th 2020.</p>
```

Comment:
```suggestion
                <p class="card-caption">The 2020 OME Community Meeting will be held remotely from May 26th to May 28th 2020.</p>
```
Can we remove this? ;)

Reply:
ufff, great catch, thanks @joshmoore

Reply:
the local docker command via jekyll somehow stopped to rebuild when the html is changed and saved. I did not understand it first, and was injecting some nonsense into the text to make sure I will see any change in the built , and then forgot to remove it.